gqlalchemy/transformations/export/graph_transporter.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2016-2022 Memgraph Ltd. [https://memgraph.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gqlalchemy.transformations.export.transporter import Transporter
+from gqlalchemy.transformations.translators.dgl_translator import DGLTranslator
+from gqlalchemy.transformations.translators.nx_translator import NxTranslator
+from gqlalchemy.transformations.translators.pyg_translator import PyGTranslator
+from gqlalchemy.memgraph_constants import MG_HOST, MG_PORT, MG_USERNAME, MG_PASSWORD, MG_ENCRYPTED, MG_CLIENT_NAME, MG_LAZY
```

Comment:
Maybe change it as following:
import gqlalchemy.memgraph_constants as mg_consts

And later you can use it as following:
mg_consts.MG_HOST

It seems less crowded on import this way, what do you think?

Reply:
Si si I agree

Reply:
@antoniofilipovic  please resolve your comments